azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -57,6 +58,7 @@ def _get_config_kwargs(**kwargs):
         "auto_connect",
         "connection_retry",
         "connection_retry_interval",
+        "model_id",
```

Comment:
is this truly a "config kwarg"? Stuff like device identification doesn't go in the pipeline config generally